helpers/PermissionManager.js

```diff
@@ -223,6 +223,17 @@ module.exports.expireAllAccessTokens = function(webstrateId, local) {
 	}
 };
 
+/**
+ * Check if configuration option is set so users must be logged in to write in a webstrate
+ * and the given user is anonymous
+ * @param  {string}   username    Username.
+ * @param  {string}   provider    Login provider (GitHub, Facebook, OAuth, ...).
+ * @return {bool}                 The configuration option is set and the user is anonymous
+ */
```

Comment:
Right about here, we tend to include either a `@private` or `@public` flag. This function isn't exported, so it'd be `@private`. But I see we aren't very consistent about this throughout the codebase, so feel free to ignore.